### PR TITLE
Limit of yens

### PR DESCRIPTION
**Pull request by mewartsbr**
I added `if()` to limit the amount of yen for when adding results calculated to not form an infinite value. Recent Javascript updates are returning an infinite value **(Symbol ∞)** that there are no methods for capturing numbers that are in that infinite value.

**Locales:**
Has added a translation of the yen limit on locales

**Screenshots:**
test

### CODE REVIEW

**Review by MysteryMS — CHANGES_REQUESTED** [review]
Can you change "this **account**..." and the respective locales to "this **amount**..."?
Makes more sense

**Comment by mewartsbr** [comment]
All locales or just `en-us`?

**Comment by MysteryMS** [comment]
All of them: "account" to "amount" and "conta" to "transferência"

**Comment by mewartsbr** [comment]
Sure, i will correct

**Review by MysteryMS — APPROVED** [review]

**Comment by mewartsbr** [comment]
Okay, locales have been fixed 😃👍 .Can close the pr ?

**Review by MysteryMS — APPROVED** [review]
Wait for @DanielaGC, she always has the final review. I'm not sure if she would like to add a limit system

**Comment by mewartsbr** [comment]
Okay, I'll be waiting.

**Review by DanielaGC — APPROVED** [review]

**Pull request merged**
